bambi/families/family.py

```diff
@@ -92,7 +110,7 @@ def __repr__(self):
 }
 
 
-def extract_family_prior(family, priors):
+def _extract_family_prior(family, priors):
```

Comment:
Is the idea that the user wont be using this method?

Reply:
Right, it is only used internally and I don't see why I a user would use it right now